server/datastore/mysql/software_titles.go

```diff
@@ -24,17 +24,21 @@ SELECT
 FROM software_titles st
 JOIN software_titles_host_counts sthc ON sthc.software_title_id = st.id
 WHERE st.id = ?
-AND sthc.team_id = 0
+AND sthc.team_id = ?
```

Comment:
i don't believe rows with 0 counts exist in `software_titles_host_counts`, so we prob need to change to a LEFT JOIN here